ENIGMAsystem/SHELL/Platforms/Win32/WINDOWSshell.cpp

```diff
@@ -0,0 +1,107 @@
+#define byte __windows_byte_workaround
```

Comment:
Please preserve the original copyright, here. If you changed more than a few lines, please add your name to it (looks like it's just the original, in which case, don't).

Reply:
@JoshDreamland i wrote execute_shell_for_output() on Windows, which is a lot of lines, and takes up more than half of the file. I also am the one who geot execute_program into a working state that doesn't crash by adding a proper waiting mechanism.